tests/cli/cli.spec.ts

```diff
@@ -56,7 +56,13 @@ select 1 as test
 
     // Compile the project using the CLI.
     const compileResult = await getProcessResult(
-      execFile(nodePath, [cliEntryPointPath, "compile", projectDir, "--json"])
+      execFile(nodePath, [
+        cliEntryPointPath,
+        "compile",
```

Comment:
shall we include vars for the `run` (dry run) case below, too? (would catch situations like the bug in this PR)